app/domain/authentication/authn_jwt/signing_key/fetch_provider_uri_signing_key.rb

```diff
@@ -13,12 +13,13 @@ def initialize(authentication_parameters:,
 
           @fetch_required_secrets = fetch_required_secrets
           @resource_class = resource_class
-          @resource_id = authentication_parameters.authn_jwt_variable_id
+          @variable_id = authentication_parameters.authn_jwt_variable_id
```

Comment:
this can go in a method instead of setting it in the init method. or using def_delegators

Reply:
Done